app/src/test/java/org/mozilla/tv/firefox/ext/LocaleKtTest.kt

```diff
@@ -8,6 +12,8 @@ import org.junit.Test
 import org.mozilla.tv.firefox.architecture.KillswitchLocales
 import java.util.*
 
+private const val LOCALE_EXT_PACKAGE = "org.mozilla.tv.firefox.ext.LocaleKt"
```

Comment:
nit: this is a (generated) class name, not a package.

Reply:
Whoops